hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java

```diff
@@ -125,21 +122,16 @@ public ManagementCenterService(HazelcastInstanceImpl instance) {
         this.timedMemberStateFactory = new TimedMemberStateFactory(instance);
         this.identifier = newManagementCenterIdentifier();
 
-        registerListeners();
+        if (this.managementCenterConfig.isEnabled()) {
+            this.instance.getCluster().addMembershipListener(new ManagementCenterService.MemberListenerImpl());
```

Comment:
You don't need to add the prefix 'ManagementCenterService.' since you are already in this class.